List-of-Free-Code-Camp-city-based-Campsites.md

```diff
@@ -662,7 +662,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
 - South Korea
   - [Busan](https://www.facebook.com/groups/free.code.camp.busan/)
   - [Daegu](https://www.facebook.com/groups/free.code.camp.your.daegu/)
-  - [Gwangju](https://www.facebook.com/groups/204942919854198/)
+  - [Gwangju](https://www.facebook.com/groups/GwangjuCodeCano/)
   - [Jeonju](https://www.facebook.com/groups/free.code.camp.jeonju/)
```

Comment:
We should encorage them to follow the naming convention of `free.code.camp.Gwengju.Code.Cano` or whatever that translates to.